course7/homeworks/patrickh/ls/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.0.0)
+set(CMAKE_CXX_STANDARD 14)
+project(ls)
+
+set(Boost_USE_STATIC_LIBS        ON)
+set(Boost_USE_MULTITHREADED      ON)
+set(Boost_USE_STATIC_RUNTIME    OFF)
+find_package(Boost COMPONENTS filesystem REQUIRED)
+
+if(Boost_FOUND)
```

Comment:
If you requested Boost as a REQUIRED in the ```find_package``` above, then there is no point checking if it was found.